.github/workflows/continuous-integration-workflow.yml

```diff
@@ -28,10 +28,6 @@ jobs:
       run: |
         pytest --cov=econsa --cov-report=xml
 
-        # We want to make sure proper formatting of the notebooks.
-        flake8-nb; [ $? -eq 0 ] || exit 1
-        black-nb --check .; [ $? -eq 0 ] || exit 1
-
```

Comment:
This is  moved to `.pre-commit-config.yaml` (L21 & L39), which prevents formatting problems from failing the run.